web-api/src/storage.rs

```diff
@@ -13,6 +13,8 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 pub(crate) mod elastic;
+#[cfg(test)]
```

Comment:
we can use a different feature gate, i just put it behind `test` because i didn't know how/where the individual pieces will be put together later. should we introduce a feature right now? we could also introduce one once the code is used somewhere?